docs/reference/search/request/docvalue-fields.asciidoc

```diff
@@ -11,13 +11,38 @@ GET /_search
     "query" : {
         "match_all": {}
     },
-    "docvalue_fields" : ["test1", "test2"]
+    "docvalue_fields" : [
+        {
+            "field": "my_ip_field", <1>
+            "format": "use_field_mapping" <2>
+        },
+        {
+            "field": "my_date_field",
+            "format": "epoch_millis" <3>
+        }
+    ]
 }
 --------------------------------------------------
 // CONSOLE
+<1> the name of the field
+<2> the special `use_field_mapping` format tells Elasticsearch to use the format from the mapping
```

Comment:
Could we not make the `format` field optional and indicate to use the field mapping by the option not being present? This would be in line with other places we support `format` such as in the date range query and date range aggregation

Reply:
This is the goal. The issue is that if we do this in 6.4 this will be a breaking change. For instance today date fields are returned as the output of Joda's ReadableInstant#toString(). I plan to make using the format from mappings the default in a follow-up PR that will only target 7.0.

Reply:
ok great, thanks for clarifying